docs/manual/working/scalaGuide/main/sql/slick/PlaySlickMigrationGuide.md

```diff
@@ -119,4 +119,4 @@ For a full example, have a look at [this sample projet](https://github.com/playf
 
 Previous versions of Play Slick included a DDL plugin which would read your Slick tables definitions, and automatically creates schema updates on reload. While this is an interesting and useful feature, the underlying implementation was fragile, and and relied on the assumption that your tables would be accessible via a module (i.e., a Scala `object`). This coding pattern was possible because Play Slick allowed to import the Slick driver available via a top-level import. However, because support for [automatic detection of the Slick driver](#Automatic-Slick-driver-detection) was removed, you will not declare a top-level import for the Slick driver. This implies that Slick tables will no longer be accessible via a module. This fact breaks the assumption made in the initial implementation of the DDL plugin, and it's the reason why the feature was removed.
 
-The consequence of the above is that you are in charge of creating and managing your project's database schema. Therefore, whenever you make a change a Slick table in the code, make sure to also update the database schema. If you find it tedious to manually keep in sync your database schema and the related table definition in your code, you may want to have a look at the code generation feature available in Slick.
\ No newline at end of file
+The consequence of the above is that you are in charge of creating and managing your project's database schema. Therefore, whenever you make a change a Slick table in the code, make sure to also update the database schema. If you find it tedious to manually keep in sync your database schema and the related table definition in your code, you may want to have a look at the code generation feature available in Slick.
```

Comment:
@dotta no, I don't know why github marked that line as changed. This PR is supposed to be the simplest of typo fixes ('projet' -> 'project'), which is why I did it via the Github edit interface on the web. Does that make sense?